mdx/api/index.mdx

```diff
@@ -682,6 +682,39 @@ Easily wrap and unwrap between ETH and WETH by requesting a swap quote by settin
 ```
 </CodeTabs>
 
+#### Ensure RFQ-T liquidity
+
+RFQ-T liquidity can only be accessed if the request supplies a taker address, a permissioned API key via header `0x-api-key`, and the query parameter `intentOnFilling=true`.  For more details see the guide [Understanding RFQ-T and the 0x API](/docs/guides/understanding-rfqt-0x-api).  Also, ensure that `Native` sources have not been excluded.
```

Comment:
Just to check, do we need a "What is RFQT liquidity" chapter, or can we assume it somehow? my understanding is that this documentation will be rendered on the main webpage, therefore we may need an introduction.